packages/web3-react/src/constants/supportedConnectors.tsx

```diff
@@ -100,8 +78,7 @@ if (typeof window != 'undefined') {
         qrCode: <Logos.WalletConnect background={true} />
       },
       logoBackground: 'var(--ck-brand-walletConnect)',
-      scannable: true,
-      defaultConnect: () => {}
```

Comment:
Since this prop is optional, and since there's nothing to initialize with, I removed it to solve the error